src/api.rs

```diff
@@ -103,6 +103,8 @@ pub struct InscriptionRecursive {
   pub charms: Vec<Charm>,
   pub content_type: Option<String>,
   pub content_length: Option<usize>,
+  #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
```suggestion
```
This is a good idea but in order to stay consistent with the other Option fields we should remove it